resources/ontology/questions/hobbyQuestion.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
why do you have separate file for each question? they can all be in the same file with extra field for intent.
is there any particular reason you place them separately?